docs/pages/repo/docs/core-concepts/caching/environment-variable-inputs.mdx

```diff
@@ -84,38 +84,41 @@ Exclusion patterns apply only to the set of variables matched via inclusions.
 
 ## Loose & Strict Environment Modes
 
-Turborepo offers two environment variable modes for tasks: loose mode and strict mode. This controls which environment variables are made available to each task at execution time.
+Turborepo offers two different modes for handling environment variable in tasks: `loose` mode and `strict` mode. The mode controls which environment variables are made available to each task at execution time.
 
 ### Loose Mode
 
-By default, and when `turbo` is invoked with `--env-mode=loose`, all environment variables from the machine are made available to every single task. This is described as "loose" environment mode. This ensures the greatest compatibility while accepting some risk that a task will implicitly have access to an unspecified environment variable. A user may configure a particular environment variable that results in task behavior change, but, because it was not specified inside of `turbo.json`, the created cache artifact may not be suitable for sharing.
+Loose mode, the default behavior, does not filter any environment variables. Therefore, all environment variables from the machine are made available to every single task. This ensures the greatest compatibility while accepting some risk that a task will implicitly have access to an unspecified environment variable. A user may run a task with a different value of an environment variable, but incorrectly see FULL TURBO because the environment variable was not configured in `turbo.json`.
 
 ### Strict Mode
 
-In strict mode, when `turbo` is invoked with `--env-mode=strict`, only [important system environment variables](#system-environment-variables) and environment variables enumerated inside of `turbo.json` will be made available to a task. There are four fields that enable specification of environment variables:
+Strict mode, which can be enabled with `turbo --env-mode=strict`, filters out environment variables.
 
-- **Hashed Environment Variables:** `globalEnv` and `env`
-- **Unhashed Environment Variables:** `globalPassThroughEnv` and `passThroughEnv`
+Only [important system environment variables](#system-environment-variables) and environment variables configured inside of `turbo.json` will be made available to a task.
+
+If you want an environment variable to be part of the cache key, use [Hashed Environment Variables](#hashed-environment-variables).
+
+If you want an environment variable to be part of the cache key, use [Unhashed Environment Variables](#unhashed-environment-variables).
```

Comment:
```suggestion
If you **do not** want an environment variable to be part of the cache key, use [Unhashed Environment Variables](#unhashed-environment-variables).
```

Reply:
Wow that would have been a bad typo, nice catch!